java/robox-slicer-extension/robox-slicer-flow/src/main/java/com/roboxing/slicerextension/flow/Slicer.java

```diff
@@ -68,6 +68,8 @@ public JSONObject getSlicerConfig() {
     }
 
     public void postProcess(File inputGCode, File resultGCode) throws IOException{
+        String newLine = System.lineSeparator();
```

Comment:
I've pulled your branch and it seems that it clashes with newLine below (line 125). I would suggest this being moved up to private static final String NEW_LINE as it won't change during the execution of our code. So you can replace newLine everywhere with 'NEW_LINE'...